tools/diffscripts/diffapi.ps1

```diff
@@ -0,0 +1,203 @@
+# Copyright (c) Microsoft. All rights reserved.
+# Licensed under the MIT license. See LICENSE file in the project root for full license information.
+
+<#
+
+.SYNOPSIS
+Microsoft Azure IoT SDK Release comparison script
+
+.DESCRIPTION
+This script is used to compare the current candidate for release to the previous version that is in the iot-sdk-internals 
+
+Parameters:
+    -AsmToolPath: The path to the compiled AsmDiff tool found in the dotnet arcade (ex: c:\tools\asmdifftool)
```

Comment:
this looks like a prereq to using this script, I think a readme might help line out the steps required to be carried out.

Reply:
Was planning on putting this in our internal one note. Will add some guidance in a readme as well